test/e2e/cli_test.go

```diff
@@ -1254,7 +1254,7 @@ func (s *CLISuite) TestSynchronizationWfLevelMutex() {
 		RunCli([]string{"get", "synchronization-wf-level-mutex"}, func(t *testing.T, output string, err error) {
 			assert.Contains(t, output, "Pending")
 		}).
-		WaitForWorkflow().
+		WaitForWorkflow(fixtures.ToBeCompleted, 120*time.Second).
```

Comment:
this is really too long for any e2e test to run - does it typically take 2m?

Reply:
In my local, each workflow takes 50s to 1.10m based on the resource in k3d.  that's why I  configured two conditions `complete` and timeout.